es-ds-docs/pages/organisms/zip-code-form.vue

```diff
@@ -261,6 +262,29 @@ const slotTableRows = [
             </es-row>
         </div>
 
+        <div class="mb-500">
+            <h2>context Message Example</h2>
```

Comment:
Can we correct the capitalization to sentence case per our [typography guidelines for headings](https://new.design.energysage.dev/atoms/typography) like "Context message example"?